css/selectors/media/sound-state.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<title>Sound State: the :muted and :volume-locked pseudo-classes</title>
+<link
+  rel="help"
+  href="https://w3c.github.io/csswg-drafts/selectors/#sound-state"
+/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+  <video width="300" height="300" muted loop></video>
+  <script type="module">
+    test((t) => {
```

Comment:
```suggestion
    // Unfortunately, we can't test the volume-locked state because it's not
    // possible to lock the volume of a video element with JS.
    
    test((t) => {
```